fabric-rendering-v1/src/testmodClient/java/net/fabricmc/fabric/test/rendering/client/HudTests.java

```diff
@@ -98,6 +105,19 @@ private static void renderAfterSubtitles(DrawContext context, RenderTickCounter
 		context.drawCenteredTextWithShadow(MinecraftClient.getInstance().textRenderer, "6. This yellow rectangle should block the player list.", context.getScaledWindowWidth() / 2, 0, Colors.WHITE);
 	}
 
+	private static HudElement replaceExperienceLevel(HudElement hudElement) {
+		return (context, tickCounter) -> {
+			if (!shouldRender) {
+				hudElement.render(context, tickCounter);
+			} else {
+				context.getMatrices().pushMatrix();
+				context.getMatrices().translate(0.0F, -3.0F);
+				hudElement.render(context, tickCounter);
+				context.getMatrices().popMatrix();
+			}
+		};
+	}
```

Comment:
This test file is in a specific format and is automated. I think we need to either change or remove this.